porting justin's edits into GH (#253)
jaybee84/ml-in-rd: content/02.intro.md

In the absence of known disease subtypes, unsupervised ML approaches can be applied to gene expression data to identify groups of samples with similar patterns of molecular states or pathway activity.  [@doi:10.1158/0008-5472.CAN-08-2100].
Unsupervised approaches can also extract combinations of features (e.g., genes) that may describe a certain cell type or pathway.
See Box 1 for more examples of how ML can be used in rare disease research.

While ML can be a useful tool, there are challenges in applying ML to rare disease datasets. 
ML methods are generally most effective when using large datasets; analyzing high dimensional biomedical data such as gene expression with many thousands of features from rare disease datasets that typically contain relatively few samples is challenging[@https://www.fda.gov/media/99546/download; @doi:10.1186/s13023-020-01424-6].
Small sample datasets tend to lack statistical power and magnify the susceptibility of ML to misinterpretation and unstable performance. 
With insufficient data, an unsupervised model will fail to identify patterns that are useful for biological discovery. 

> In the case of supervised models, modelsthey can be adversely impact if sample labels are uncertain or contain “label-noise”. [@doi:10.1093/jamia/ocw028] 

Reviewer: ```suggestion
In the case of supervised models, they can be adversely impact if sample labels are uncertain or contain “label-noise”. [@doi:10.1093/jamia/ocw028] 
```

> Datasets with high label-noise decrease prediction accuracy and necessitate larger sample sizes during model training [@doi:10.1109/tnnls.2013.2292894].

Author: ```suggestion
Datasets with high label-noise decrease prediction accuracy and necessitate larger sample sizes during model training [@doi:10.1109/tnnls.2013.2292894] (See Box 2).
```

Author: In response to Justin's comment: This is at the first time I’ve seen “training” mentioned. Should be defined

> Datasets with high label-noise decrease prediction accuracy and necessitate larger sample sizes during model training [@doi:10.1109/tnnls.2013.2292894].

Reviewer 2: We could perhaps be more specific about what we mean by training here in the sentence itself:

```suggestion
Datasets with high label-noise decrease prediction accuracy and necessitate larger sample sizes during the process by which models learn patterns that distinguish samples in different classes [@doi:10.1109/tnnls.2013.2292894] (_model training_; Box 2).
```

That rewording assumes we are talking about classification problems here.

> Rare disease datasets often come with significant label-noise (e.g., silver standard datasets) due to limited understanding of the underlying biology or evolving clinical classifications. 

Author: Justin: Not sure I agree with this generalization

Author: ```suggestion
Many rare disease datasets may come with significant label-noise (e.g., silver standard datasets) due to limited understanding of the underlying biology or evolving clinical classifications. 
```

Reviewer: I am wondering which part justin takes issue with - that most datasets come with this, or that it's "due to limited understanding of the underlying biology or evolving clinical classifications."

Reviewer: What if we reframed this with an example? 

https://www.medrxiv.org/content/10.1101/2020.07.06.20147322v1.full

```suggestion
Rare disease-relevant datasets, such as electronic healthcare records [@doi:10.1101/2020.07.06.20147322], often come with significant label-noise (e.g., silver standard datasets). 
```

Author: @allaway One might say that label noise is abundant in EHR, even for diseases that are well characterized or are not rare. Just the nature of RWD.

I think here we wanted to highlight that for rare disease subtypes, the distinction between labels of the subtypes are fuzzy due to lack of complete understanding of the disease mechanism. So even if a sample is labeled a specific tumorType like PNF, it could actually be an Atypical NF (hence a silver standard dataset).

Reviewer 2: > I think here we wanted to highlight that for rare disease subtypes, the distinction between labels of the subtypes are fuzzy due to lack of complete understanding of the disease mechanism. So even if a sample is labeled a specific tumorType like PNF, it could actually be an Atypical NF (hence a silver standard dataset).

Agree, and to take things a bit further: part of what I think we're getting at with how the text is currently written is that if you have to use every cohort that is available, and those cohorts were collected a different points in time, you can imagine a situation where the labels are not totally comparable.

Reviewer 2: ```suggestion
Rare disease datasets often come with significant label-noise.
For example, if classifications of rare disease subtypes evolve over time, researchers constructing datasets for ML research may find that cohorts collected at different time periods do not have comparable labels.
```

Notes:

* Might be missing an adverb that modifies "comparable" to communicate _degree_
* Is missing the point about (citation of?) silver standard datasets
* Not necessarily specific to rare diseases as currently written
